src/Sylius/Bundle/ResourceBundle/EventListener/ResourceDeleteSubscriber.php

```diff
@@ -161,4 +155,14 @@ private function isHtmlRequest(Request $request)
     {
         return 'html' === $request->getRequestFormat();
     }
+
+    /**
+     * @param $resource
+     *
+     * @return string
+     */
+    protected function castResourceToString($resource)
+    {
+        return (string) new StringConverter($resource);
```

Comment:
I can't find `_toString` static method in `Coduo\ToString\StringConverter` class so do you mean any other library?